google/cloud/spanner_v1/transaction.py

```diff
@@ -80,9 +80,14 @@ def _make_txn_selector(self):
         self._check_state()
         return TransactionSelector(id=self._transaction_id)
 
-    def begin(self):
+    def begin(self, read_only=False):
         """Begin a transaction on the database.
 
+        :type read_only: bool
+        :param read_only:
+            (Optional) If True, ReadOnly transaction type will be
+            begun, ReadWrite otherwise.
```

Comment:
```suggestion
            (Optional) If True, a read-only transaction with strong timestamp bound will be started.
            Otherwise a read/write transaction will be started.
```